File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.js
function solution(arr1, arr2) {
    if(arr1.length !== arr2.length) {
        return arr1.length > arr2.length ? 1 : -1;
    }
    
    let arr1_sum = 0;
    let arr2_sum = 0;
    
    arr1.forEach(n => arr1_sum += n);
    arr2.forEach(n => arr2_sum += n);
    
    return arr1_sum === arr2_sum ? 0 : arr1_sum > arr2_sum ? 1 : -1;
}

File: 프로그래머스/0/181856. 배열 비교하기/배열 비교하기.py
def solution(arr1, arr2):
    len1 = len(arr1)
    len2 = len(arr2)
    
    if len1 != len2:
        return 1 if len1 > len2 else -1
    
    return 1 if sum(arr1) > sum(arr2) else -1 if sum(arr2) > sum(arr1) else 0
        